apps/mobile/src/components/EditScreenInfo.tsx

```diff
@@ -7,31 +7,15 @@ import { Text, View } from './Themed'
 
 export default function EditScreenInfo({ path }: { path: string }) {
     return (
-        <View>
+        <View style={styles.screen}>
             <View style={styles.getStartedContainer}>
+                <Text style={styles.title}>How to get a Proof of Networking</Text>
                 <Text style={styles.getStartedText} lightColor='rgba(0,0,0,0.8)' darkColor='rgba(255,255,255,0.8)'>
-                    Open up the code for this screen:
-                </Text>
-
-                <View
-                    style={[styles.codeHighlightContainer, styles.homeScreenFilename]}
-                    darkColor='rgba(255,255,255,0.05)'
-                    lightColor='rgba(0,0,0,0.05)'>
-                    <MonoText>{path}</MonoText>
-                </View>
-
-                <Text style={styles.getStartedText} lightColor='rgba(0,0,0,0.8)' darkColor='rgba(255,255,255,0.8)'>
+                    Scan the QR code in your new acquaintance's mobile app:{' '}
```

Comment:
I'm not sure we are meant to make changes here - I believe this is just a placeholder file meant to show message of what file you should be editing to update any particular screen ( in this case should be QR )

Reply:
I removed the placeholder component.